pkg/cmd/pr/merge/merge.go

```diff
@@ -367,21 +386,21 @@ func (m *mergeContext) deleteLocalBranch() error {
 			return err
 		}
 
-		if git.HasLocalBranch(targetBranch) {
-			if err := git.CheckoutBranch(targetBranch); err != nil {
+		if git.HasLocalBranch(m.pr.BaseRefName) {
```

Comment:
the variable `targetBranch` might be clearer here, not sure if it's worth keeping it as
`targetBranch := m.pr.BaseRefName`

Reply:
Yeah let's do that!

Reply:
done!